### PR TITLE
compile warnings resolved

### PR DESCRIPTION
**Pull request by cjakeman**
mostly by removing redundant debug statements
https://bugs.launchpad.net/or/+bug/1889790

### CODE REVIEW

File: Source/RunActivity/Viewer3D/Popups/HelpWindow.cs [hunk 1]
```diff
@@ -66,6 +66,9 @@ public class HelpWindow : Window
         List<TabData> Tabs = new List<TabData>();
         int ActiveTab;
 
+        private enum FuelTypes { Coal, DieselOil, Kwhr }
+        private FuelTypes FuelType;
```
[the remark this answers is not in the document]
Comment (YoRyan) [reply Source/RunActivity/Viewer3D/Popups/HelpWindow.cs:70]: Looking more closely, the `if {} if {}` branching tests the value of a `TrainCar.EngineTypes` enum, so there is indeed a 1:1 mapping between an engine its fuel source. (Or at least this code already assumes so.)

File: Source/Orts.Formats.Msts/ActivityFile.cs [hunk 2]
```diff
@@ -450,13 +450,19 @@ public class StartTime {
     public class Duration {
         int Hour;
         int Minute;
-        int Second;
+        private int Second; // private as currently never assigned to
```
[the remark this answers is not in the document]
Comment (cjakeman) [reply Source/Orts.Formats.Msts/ActivityFile.cs:453]: I tried that and the compiler still gave a warning.

[the remark this answers is not in the document]
Comment (YoRyan) [reply Source/Orts.Formats.Msts/ActivityFile.cs:453]: Interestingly enough, Visual Studio doesn't even give me that warning with the current code.

Humor me and see if this patch satisfies your compiler. (I apologize for the copy-paste job, but GitHub is refusing to attach it.)

```diff --git a/Source/Orts.Formats.Msts/ActivityFile.cs b/Source/Orts.Formats.Msts/ActivityFile.cs
index 0c56e2df4..2f4db271f 100644
--- a/Source/Orts.Formats.Msts/ActivityFile.cs
+++ b/Source/Orts.Formats.Msts/ActivityFile.cs
@@ -450,11 +450,12 @@ public class StartTime {
     public class Duration {
         int Hour;
         int Minute;
-        private int Second; // private as currently never assigned to
+        int Second;
 
         public Duration(int h, int m) {
             Hour = h;
             Minute = m;
+            Second = 0;
         }
 
         public Duration(int h, int m, int s) {
@@ -468,6 +469,7 @@ public class Duration {
             Hour = stf.ReadInt(null);
             Minute = stf.ReadInt(null);
             stf.MustMatch(")");
+            Second = 0;
         }
 
         public int ActivityDuration()

```

**Comment by YoRyan** [comment]
I'm also seeing one last warning related to the `ActEditor.ToolClicked` variable. It's being set, but it's never accessed. I assume it's related to some incomplete feature, so here's another [patch](https://github.com/openrails/openrails/files/5135013/fixtoolclicked.diff.txt) to comment this variable out.

File: Source/Orts.Formats.Msts/ActivityFile.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (cjakeman) [reply Source/Orts.Formats.Msts/ActivityFile.cs:453]: I'll try that tomorrow.

Best wishes,

Chris

[the remark this answers is not in the document]
Comment (cjakeman) [reply Source/Orts.Formats.Msts/ActivityFile.cs]: Hi Ryan,

Strangely, the compiler was satisfied with the first Second = 0; and didn't need the second.

Thanks,

Chris

**Comment by cjakeman** [comment]
> I'm also seeing one last warning related to the `ActEditor.ToolClicked` variable. It's being set, but it's never accessed.

I've applied those changes too.

Chris

**Comment by YoRyan** [comment]
It appears recently merged PR #268 introduced an additional compiler warning.

Visual Studio says the `ManualMaxBrakeForceN` [variable](https://github.com/openrails/openrails/blob/1848b2b7ed317385fcab617a994c9a16c2e58c09/Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/ManualBraking.cs#L47) is not used by any code. Any guidance here, @peternewell?

**Review by YoRyan — APPROVED** [review]
For the moment, approving. We'll deal with the manual braking warning later.

**Pull request merged**